web/src/lib/components/ItemView.svelte

```diff
@@ -128,21 +128,21 @@
 </script>
 
 <MetaTags
-    title={(serverLoadedItem?.title ?? "Plebeian Market item").substring(0,69)}
-    description={(serverLoadedItem?.description ?? import.meta.env.VITE_PM_DESCRIPTION).substring(0,199)}
+    title={serverLoadedItem?.getShortTitle()}
+    description={serverLoadedItem?.getShortDescription()}
     openGraph={{
         site_name: "Plebeian Market",
```

Comment:
Can we also get site_name from somewhere? Ideally from .env.XXX (`VITE_SITE_NAME`), but a constant in utils.ts would also be better than this...

Reply:
Correct!